### PR TITLE
Fix mobile audio

### PR DESCRIPTION
**Pull request by defnotryan**
The root issue was that in iOS, audio elements cannot autoplay, and the
play method must be called during a user interaction event.

### CODE REVIEW

**Comment by defnotryan** [comment]
I only have ready access to iOS so I wasn't able to test on Android. Would be interested to see results if anyone has Android.

**Comment by calebwoods** [comment]
Tested on an Android tablet and seems to work well.

File: js/components/AudioPlayer.react.js [hunk 1]
```diff
@@ -7,26 +7,53 @@ import { connect } from 'react-redux';
...
+  }
+
+  renderPauseButtonAndAudio() {
+    const { dispatch } = this.props;
+    return (
+      <span>
+        <button className="pauseAudio" onClick={() => {dispatch(asyncPauseAudio())}}>
+          <i className="fa fa-pause" title="pause"></i>
+        </button>
+        { this.renderAudioElement() }
```
Comment (calebwoods) [inline js/components/AudioPlayer.react.js]: Is there a reason we couldn't just always have the `<audio>` element rendered with the button dispatchings controls?

**Comment by defnotryan** [comment]
@calebwoods Like this? I wasn't sure where best place was to put the calls to `audio.play()` and `audio.pause()`; let me know if they should go somewhere else.

**Comment by calebwoods** [comment]
@defnotryan I think that location makes sense.  Tested on a couple of devices and seems to work well.

**Comment by danahartweg** [comment]
The code looks good to me. 

:+1

**Comment by defnotryan** [comment]
Resolved some conflicts with incoming changes from keyboard navigation branch.

**Pull request merged**